spec/features/admin/admin_deletes_user_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+feature 'admin deletes user', %{
+  As an admin
+  I want to be able to delete a user
+  so that I may remove users who no longer need access
+  } do
+
+    # Acceptance Criteria:
+    # [X] I must be able to delete a user
+    # [X] I'm asked to confirm deleting a user
+    # [X] I must be an admin to delete a user
+    # [X] I'm notified on success
+
+  let(:admin) { FactoryGirl.create(:user, role: 'admin') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.